include/logging/log_core.h

```diff
@@ -357,9 +357,21 @@ static inline char z_log_minimal_level_to_char(int level)
 	} \
 } while (false)
 
+#define Z_LOG_DO_ACTION_ONCE(code)	\
+({ \
+	static _bool done_once = false; \
```

Comment:
Sorry that must be a typo

Reply:
Might consider changing the name to something like `log_done_once` so it will be less confusing when someone is going through the symbol table.

Reply:
Thanks! Indeed it was a confusing name. I moved to sys_utils (since it's not log-specific) and renamed as ```DO_FIRST_N``` so that it's more clear that it just silences a piece of code after a given ```n```. Does that sound better?